src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const add = (a) => {
```

Comment:
```suggestion
  const add = (num) => {
```

Better to use more meaningful names